requirements.txt

```diff
@@ -1,5 +1,6 @@
 annotated-types==0.7.0
 asgiref==3.8.1
+async-timeout==5.0.1
```

Comment:
I am not seeing any usage of this library, any specific reason to add it? Else remove unnecessary requirements.